carma/launch/guidance.launch.py

```diff
@@ -316,6 +316,59 @@ def generate_launch_description():
         ]
     )
 
+    trajectory_follower_container = ComposableNodeContainer(
+        package='carma_ros2_utils',
+        name='trajectory_follower_container',
+        executable='carma_component_container_mt',
+        namespace=GetCurrentNamespace(),
+        composable_node_descriptions=[
+            ComposableNode(
+                package='trajectory_follower_nodes',
+                plugin='autoware::motion::control::trajectory_follower_nodes::LatLonMuxer',
+                name='latlon_muxer_node',
+                extra_arguments=[
+                    {'use_intra_process_comms': False},
+                    {'--log-level' : GetLogLevel('latlon_muxer', env_log_levels) }
+                ],
+                parameters=[
+                    {'timeout_thr_sec':0.5}
+                ]
```

Comment:
If this node also has some output, input topic in the name. it should probably be remappaed to drop those as mentioned in my later comments

Reply:
The output of this node are subscribed to by trajectory follower wrapper, and the remapping is implemented there